client/ayon_xstudio/plugins/load/open_in_xstudio.py

```diff
@@ -94,9 +98,7 @@ def load(
             # we output the seq path with hashes.
             idxs = list(seq.indexes)
             pad = "#" * (seq.padding + len(str(idxs[0])))
```

Comment:
```suggestion
            pad = "#" * seq.padding
```

Here is what was happeing to me on some sequences:
```
path : C:\projects\AY01_VFX_demo\shots\sq060\sh0010\publish\render\renderCompositingMain\v001\a01vfxd_sh0010_renderCompositingMain_v001.0999.exr  ]
>>> [  pattern : \.(?P<index>(?P<padding>0*)\d+)\.\D+\d?$  ]
>>> [  collections : [<Collection "a01vfxd_sh0010_renderCompositingMain_v001.%04d.exr [999-1007]">]  ]
>>> [  1first_image : None  ]
>>> [  idxs : [999, 1000, 1001, 1002, 1003, 1004, 1005, 1006, 1007]  ]
>>> [  seq.padding : 4  ]
>>> [  pad : #######  ]
>>> [  2first_image : a01vfxd_sh0010_renderCompositingMain_v001.#######.exr=999-1007  ]
>>> [  Opening xStudio with : C:\projects\AY01_VFX_demo\shots\sq060\sh0010\publish\render\renderCompositingMain\v001\a01vfxd_sh0010_renderCompositingMain_v001.#######.exr=999-1007  ]
```

Reply:
I think it was meant as this...
```suggestion
            pad = "#" * max(seq.padding + len(str(idxs[-1])))
```